core/node/api_server/src/web3/namespaces/eth.rs

```diff
@@ -695,6 +695,8 @@ impl EthNamespace {
             base_fee_per_gas,
             gas_used_ratio,
             reward,
+            base_fee_per_blob_gas: vec![],
+            blob_gas_used_ratio: vec![],
```

Comment:
We are adding two new fields to the `eth_feeHistory` response on our API as well.
This is an implication of using the same types both internally and for API.

Is this OK? I would think so, given that adding new fields is not breaking, and this actually makes our API more compatible with L1.

Reply:
that's what I meant in my initial comment, we could skip the serialization for them as a result we will not change the behaviour of our api